cmd/api-linter/cli.go

```diff
@@ -105,14 +111,33 @@ func runCLI(rules lint.Rules, configs lint.Configs, args []string) error {
 				case "json":
 					marshal = json.Marshal
 				}
-				b, err := marshal(problems)
+				b, err := marshal(lintResponses)
 				if err != nil {
 					return err
 				}
-
 				if _, err = w.Write(b); err != nil {
 					return err
 				}
+
+				if c.BoolT("summary") {
+					summary, longestRuleLen := createSummary(lintResponses)
+					colOneFormat, colTwoFormat := "%-"+strconv.Itoa(max(longestRuleLen+5, 25))+"s", "%25s"
```

Comment:
Why 25? So total width for both columns is 50? Is that "normal" for a terminal width?

When moving all this logic into summary_cli.go, consider defining a var like DEFAULT_SUMMARY_COL_WIDTH to avoid magic numbers.